src/summary/history/conditions-card.component.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import style from "./conditions-card-style.css";
+import dayjs from "dayjs";
+import SummaryCard from "../cards/summary-card.component";
+import SummaryCardRow from "../cards/summary-card-row.component";
+import SummaryCardRowContent from "../cards/summary-card-row-content.component";
+import { match } from "react-router";
+import { performPatientConditionSearch } from "./conditions.resource";
+
+export default function ConditionsCard(props: ConditionsCardProps) {
+  const [patientConditions, setPatientConditions] = React.useState(null);
+
+  React.useEffect(() => {
+    const abortController = new AbortController();
+    performPatientConditionSearch(
+      props.currentPatient.identifier[0].value,
+      abortController
+    )
+      .then(condition => setPatientConditions(condition))
+      .catch(error => {
+        throw error;
+      });
+  });
+
+  return (
+    <SummaryCard name="Conditions" match={props.match}>
+      <div className={style.conditionHeader}>
+        <div>Active Conditions</div>
+        <div>Since</div>
+      </div>
+      {patientConditions &&
+        patientConditions.total > 0 &&
+        patientConditions.entry.map(condition => {
+          return (
+            <SummaryCardRow key={condition.resource.id} linkTo="/">
+              <SummaryCardRowContent>
+                <div className={style.conditionsRow}>
```

Comment:
Hmm let's switch this over to [HorizontalLabelValue](https://github.com/openmrs/openmrs-esm-patient-chart/pull/2/files#diff-019b0ec6d0108367543d9587070d3a91) once #2 is merged. That will remove the need for a lot of the css you've written, i think